### ISSUE TITLE
Move format data migration out of migration files

### ISSUE THREAD

**Issue by sirex** [atviriduomenys/katalogas#281]
Move format fixing migration out of migrations into a separate script, because this migration take very long time to complete.

https://github.com/atviriduomenys/katalogas/blob/9349239ae43534b87ef2a8a0f28ec97033c0079d/vitrina/resources/migrations/0007_files_fix.py#L193

It would be great if requests would be cached to a specified locations, so that running it repeatedly, requests, that were already cached, would read data from the cache. For caching you can use [requests-cache](https://pypi.org/project/requests-cache/) library.

See other scripts in `scripts/` directory.

Also, check other migrations if there are other slow migrations, maybe they should be moved to a script too?

Basically we need to fix migrations, to make then run fast.
